tests/serverpod_test_server/test_integration/database_operations/transactions/transaction_isolatation_level_test.dart

```diff
@@ -0,0 +1,292 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
+
+  tearDown(() async {
+    await SimpleData.db.deleteWhere(session, where: (t) => Constant.bool(true));
+  });
+
+  group(
+      'Given read committed transaction isolation level and single row in database',
+      () {
+    var settings = TransactionSettings(
+      isolationLevel: IsolationLevel.readCommitted,
+    );
+
+    test(
+        'when row is modified after first statement in transaction'
+        'then transaction observes the updated value.', () async {
+      var testData = await SimpleData.db.insertRow(
+        session,
+        SimpleData(num: 1),
+      );
```

Comment:
Suggestion: Move this out to `group` level to match the Given statement.

Reply:
Good catch! ⭐

Reply:
787086293005dec16c4d270ae70baf326e0a69da